src/stylesheets/components/_highlight.scss

```diff
@@ -32,6 +32,10 @@
   }
 }
 
+.cookie {
+  display: none;
```

Comment:
Should this be done the other way round - display:block by default - for if I don't have JS turned on?

Reply:
Needs to be hidden by default to prevent the banner appearing then being hidden, but have changed this for only users with JS turned on.